src/components/SarcophagusCreate/ArchaeologistSelect/ArchaeologistTableRow.tsx

```diff
@@ -44,6 +44,12 @@ const ArchaeolgistTableRow = ({
   const isFreeBondGreater = archaeologist.freeBond.gte(ethers.utils.parseEther(archTotal));
 
   const selectArchaeologist = () => {
+    if(isSelected) {
+      setFieldValue("bounty", "1")
+      setFieldValue("diggingFee", 6)
+      setFieldValue("address", "")
+      return
+    }
```

Comment:
@Da-Colon -- this has merge conflict -- related to `if(isDisabled) return` from the other branch